common/src/IO/Bsp29Parser.cpp

```diff
@@ -160,9 +161,19 @@ namespace TrenchBroom {
 
                 auto subReader = reader.subReaderFromBegin(static_cast<size_t>(textureOffset)).buffer();
 
+                // MipTextureReader::doReadTexture requires a texture name to be provided in the File's Path.
```

Comment:
I would prefer this function to be moved into MipTextureReader as a static function or whatever.